### ISSUE TITLE
Expose transaction information now we have the DB indexer

### ISSUE THREAD

**Issue by gordsport** [vegaprotocol/vega#6118]
# Task Overview

Now we have the db indexer we have a set of tables with all of the transactions in the database. This allowsaccess to get the last n transactions with paging etc.. 

We need to decide how we want block explorer to interact with this database, i.e. do we add some API on top of this, do we expose it through datanode, etc.

@jeremyletang @pscott31 and @dexturr - please can you discuss the best approach and add details to this issue so that we can schedule the work and unblock the current issue on the block explorer with showing transactions.

# Definition of Done
>ℹ️ Not every issue will need every item checked, however, every item on this list should be properly considered and actioned to meet the [DoD](https://github.com/vegaprotocol/vega/blob/develop/DEFINITION_OF_DONE.md).

**Before Merging**
- [ ] Create relevant for [system-test](https://github.com/vegaprotocol/system-tests/issues) tickets with feature labels
- [ ] Code refactored to meet SOLID and other code design principles
- [ ] Code is compilation error, warning, and hint free
- [ ] Carry out a basic happy path end-to-end check of the new code
- [ ] All acceptance criteria confirmed to be met, or, reasons why not discussed with the engineering leadership team
- [ ] All APIs are documented so auto-generated documentation is created
- [ ] All Unit, Integration and BVT tests are passing
- [ ] Implementation is peer reviewed (coding standards, meeting acceptance criteria, code/design quality)
- [ ] Create [front end](https://github.com/vegaprotocol/token-frontend/issues) or [console](https://github.com/vegaprotocol/console/issues) tickets with feature labels (should be done when starting the work if dependencies known i.e. API changes)

**After Merging**
- [ ] Move development ticket to `Done` if there is **NO** requirement for new system-tests
- [ ] Resolve any issues with broken system-tests
- [ ] Create [documentation](https://github.com/vegaprotocol/documentation/issues) tickets with feature labels if functionality has changed, or is a new feature

**Comment by dexturr** [comment]
Is the scope of this ticket merely for the transactions list or to retire the chain explorer API entirely?

**Comment by gordsport** [comment]
Closing as duplicate of: #6163
